pkg/apis/flowcontrol/validation/validation.go

```diff
@@ -29,6 +29,11 @@ import (
 	"k8s.io/kubernetes/pkg/apis/flowcontrol"
 )
 
+// Constants for validation.
+const (
+	FlowSchemaMaxMatchingPrecedence int32 = 10000
```

Comment:
We did this in openshift, with duplicated consts.  In the end, we preferred to use the external ones where we could.

Reply:
moved this to the external api